views/Profile.php

```diff
@@ -117,9 +117,33 @@
 
       <ul class="list-group">
         <li class="list-group-item text-muted">Activity <i class="fa fa-dashboard fa-1x"></i></li>
-        <li class="list-group-item text-right"><span class="pull-left"><strong><a href="<?php echo BASE_URL; ?>profile/edit/post/">Posts</a></strong></span><?php echo $result['totalPosts'] ?></li>
-        <li class="list-group-item text-right"><span class="pull-left"><strong>Likes</strong></span> <?php echo $result['totalLikes']; ?></li>
-        <li class="list-group-item text-right"><span class="pull-left"><strong>Shares</strong></span> <?php echo $result['totalShares'];; ?></li>
+        <li class="list-group-item text-right"><span class="pull-left"><strong><a href="<?php echo BASE_URL; ?>profile/edit/post/">Posts</a></strong></span>
+        <?php 
+          if($result['totalPosts']){
```

Comment:
Make it small  like `echo ($str1 == "Pankaj") ? "Hi Pankaj"."<br>" : "Hi There!"."<br>";`